common/persistence/EnvironmentVariableKVDriver.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ * @license GPLv2
+ *
+ */
+
+namespace oat\generis\persistence;
+
+use common_exception_NoImplementation;
+use common_persistence_KeyValuePersistence;
+use common_persistence_KvDriver;
+
+class EnvironmentVariableKVDriver implements common_persistence_KvDriver
```

Comment:
may i ask you to apply the php 7 specifics strong type and return type? 
if possible

Reply:
I've added return types, but to add scalar strong types for type hinging I need adjust interface, that would lead to bigger changes e.g. we should update all the drivers